caravel/assets/visualizations/maps/PointMap.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import ScatterPlotClusterOverlay from './ScatterPlotClusterOverlay.jsx';
+import BaseMapWrapper from './BaseMapWrapper.jsx';
+import ViewportMercator from 'viewport-mercator-project';
+import Immutable from 'immutable';
+import supercluster from 'supercluster';
+
+const DEFAULT_MAX_ZOOM = 16;
+const DEFAULT_POINT_RADIUS = 60;
+const propTypes = {
+  isDragging: React.PropTypes.bool.isRequired,
```

Comment:
I like the fact that propTypes are at the top of the file here, @ascott : should we do this everywhere else too?